packages/frontend/src/pages/ProtectedPage/index.tsx

```diff
@@ -17,6 +20,16 @@ type Props = {
 export const ProtectedPage = ({ children }: Props) => {
   const { openEdit, openToolbar } = useContext(AppBarContext);
   const { value } = useContext(EditorContext);
+  const location = useLocation();
+
+  const getRouteName = () => {
```

Comment:
Good idea! Could you move to utils so that can be reused elsewhere (e.g. EditorContext)?